<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="tbSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource_file.qrc">
      <normaloff>:/icons/res/settings.svg</normaloff>:/icons/res/settings.svg</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="tbStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource_file.qrc">
      <normaloff>:/icons/res/play.svg</normaloff>:/icons/res/play.svg</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="tbStop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource_file.qrc">
      <normaloff>:/icons/res/stop.svg</normaloff>:/icons/res/stop.svg</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="tbPause">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource_file.qrc">
      <normaloff>:/icons/res/pause.svg</normaloff>:/icons/res/pause.svg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
